src/vaadin-grid-scroll-mixin.html

```diff
@@ -413,37 +370,37 @@
 
     /** @private */
     _translateStationaryElements() {
-      if (this._edge && !this._scrollbarWidth) {
-        // Fixed mode (Tablet Edge)
-        this.$.items.style.transform =
-        this._getTranslate(-this._scrollLeft || 0, -this._scrollTop || 0);
-
-        this.$.footer.style.transform = this.$.header.style.transform =
-        this._getTranslate(-this._scrollLeft || 0, 0);
-      } else {
-        this.$.footer.style.transform = this.$.header.style.transform = this._getTranslate(0, this._scrollTop);
+      const scrollLeft = Math.max(0, this._scrollLeft); // TODO: Min
```

Comment:
What is this `TODO` comment about? Should it be removed?

Reply:
Looks like it's from more than 2 years ago 😄  Still it might be good to remove it.

Reply:
Removed.